lib/google-contacts/element.rb

```diff
@@ -289,7 +309,10 @@ def update_groups(*group_links)
     end
 
     private
-     # Evil ahead
+      def get_google_type(google_type)
```

Comment:
hello @rohandaxini 
This returns the label name which google used for `phone number`, `address`, `email address`...